### ISSUE TITLE
Нужно ли: "Создание правила на основании базового правила Xpath"

### ISSUE THREAD

**Issue by theshadowco** [theshadowco/sonar-helper#6]
Я у себя создаю правила, запрещающие использовать методы, если есть новые исправленные на основании xpath. Делаю это не руками, а запускаю скрипт, которому скармливаю набор данных для создания правила, а он создает и автоматически активирует правила в профилях. 

Если интересно, могу добавить данную возможность и в хэлпер.

**Comment by asosnoviy** [comment]
Да! это же супер фича для тех кто не может на джаве писать проверки.
